### PR TITLE
Mediator search FE

### PR DESCRIPTION
**Pull request by Mjfalahee**
# Description

Added a basic search for mediators. Added a search view, MediatorList/Card components, and Case view for a router link. 

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: client/src/components/MediatorList.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            //https://brav-staging.herokuapp.com/mediators
+            //http://localhost:8888/mediators
+            const res = await axios.get("http://localhost:8888/mediators");
+            console.log('Inside useEffect -----------------', res.data);
+            setMediators(res.data);
+        }
+
+        fetchMediators()
+        }, []);
+        //re-check when filters change eventually, but for now just mimic ComponentDidMount.
```
Comment (wurde) [inline client/src/components/MediatorList.js]: Good call, we'll need to update this when we add filters.

File: client/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://localhost:8888
```
Comment (wurde) [inline client/.env]: The `.env` file is supposed to not be in git. Seems our `.gitignore` isn't properly ignoring this local file.

Comment (Mjfalahee) [reply client/.env]: good point. I'll add .env to the ignore then?

Comment (wurde) [reply client/.env]: :100:

**Pull request merged**